ConsistencyManagerTests/ListenerTests.swift

```diff
@@ -105,6 +105,7 @@ class ListenerTests: ConsistencyManagerTestCase {
                     listener = strongListener
 
                     addListener(strongListener, toConsistencyManager: consistencyManager)
+                    waitOnMainThread()
```

Comment:
`flushMainQueueOperations`
`flushMainThreadOperations`
`flushOperationsOnMainQueue`
`flushOperationsOnMainThread`

I don't have a strong preference. I do think the extra verbosity is worth it for clarity when reading the tests.